src/components/HomeSearchBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const HomeSearchBox = () => {
+  const { t } = useTranslation('home')
+  const [searchItem, setSearch] = useState('')
```

Comment:
Since this is value of the `input`, How about naming it this way?

```suggestion
  const [searchValue, setSearchValue] = useState<string>('')
```

Reply:
I agree.